src/desktop/apps/auction_support/stylesheets/accept_conditions_of_sale_modal.styl

```diff
@@ -0,0 +1,18 @@
+
+#modal.accept-conditions-register
```

Comment:
I put these here instead of src/desktop/components/modal/index.styl where others have gone: see the confirm registration styles at
https://github.com/artsy/force/blob/master/src/desktop/components/modal/index.styl#L31